python/ray/rllib/dqn/example-cartpole.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+from ray.rllib.dqn import DQN, DEFAULT_CONFIG
+
+
+def main():
+    config = DEFAULT_CONFIG.copy()
+    config.update(dict(
```

Comment:
Note: we should make train.py importable so that it can be called from these example scripts.